2024-02-03 00:44:24,391 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-03 00:44:24,391 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Configure stats pid to 79316
2024-02-03 00:44:24,391 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/.config/wandb/settings
2024-02-03 00:44:24,391 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/settings
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_init.py:_log_setup():526] Logging user logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_004424-7xouf3jl/logs/debug.log
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_004424-7xouf3jl/logs/debug-internal.log
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x29f9911c0>
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_init.py:init():566] calling init triggers
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'model': {'n_width': 20, 'n_coupling': 32, 'input_width': 1}, 'train': {'lr': 0.001, 'batch_size': 1000, 'epochs': 40, 'device': device(type='cpu')}}
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_init.py:init():616] starting backend
2024-02-03 00:44:24,392 INFO    MainThread:79316 [wandb_init.py:init():620] setting up manager
2024-02-03 00:44:24,393 INFO    MainThread:79316 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-02-03 00:44:24,394 INFO    MainThread:79316 [wandb_init.py:init():628] backend started and connected
2024-02-03 00:44:24,396 INFO    MainThread:79316 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-03 00:44:24,396 INFO    MainThread:79316 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-03 00:44:24,396 INFO    MainThread:79316 [wandb_init.py:init():720] updated telemetry
2024-02-03 00:44:24,396 INFO    MainThread:79316 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-03 00:44:25,106 INFO    MainThread:79316 [wandb_run.py:_on_init():2254] communicating current version
2024-02-03 00:44:25,634 INFO    MainThread:79316 [wandb_run.py:_on_init():2263] got version response 
2024-02-03 00:44:25,634 INFO    MainThread:79316 [wandb_init.py:init():804] starting run threads in backend
2024-02-03 00:44:25,665 INFO    MainThread:79316 [wandb_run.py:_console_start():2233] atexit reg
2024-02-03 00:44:25,666 INFO    MainThread:79316 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-03 00:44:25,666 INFO    MainThread:79316 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-03 00:44:25,666 INFO    MainThread:79316 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-03 00:44:25,666 INFO    MainThread:79316 [wandb_init.py:init():847] run started, returning control to user process
2024-02-03 00:44:30,549 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:44:30,549 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:44:35,008 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:44:35,031 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-03 00:44:35,032 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Configure stats pid to 79316
2024-02-03 00:44:35,032 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/.config/wandb/settings
2024-02-03 00:44:35,032 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/settings
2024-02-03 00:44:35,032 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-03 00:44:35,032 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-03 00:44:35,033 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:44:35,033 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:44:35,033 INFO    MainThread:79316 [wandb_init.py:_log_setup():526] Logging user logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_004435-4ml71awd/logs/debug.log
2024-02-03 00:44:35,033 INFO    MainThread:79316 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_004435-4ml71awd/logs/debug-internal.log
2024-02-03 00:44:35,033 INFO    MainThread:79316 [wandb_init.py:init():566] calling init triggers
2024-02-03 00:44:35,033 INFO    MainThread:79316 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'model': {'n_width': 20, 'n_coupling': 32, 'input_width': 1}, 'train': {'lr': 0.001, 'batch_size': 1000, 'epochs': 40, 'device': device(type='cpu')}}
2024-02-03 00:44:35,033 INFO    MainThread:79316 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 7xouf3jl
2024-02-03 00:44:35,034 INFO    MainThread:79316 [wandb_run.py:_finish():1962] finishing run ai-gigachads/imsh/7xouf3jl
2024-02-03 00:44:35,035 ERROR   MainThread:79316 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-02-03 00:44:35,035 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:44:35,035 INFO    MainThread:79316 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-03 00:44:35,036 INFO    MainThread:79316 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-03 00:44:35,036 INFO    MainThread:79316 [wandb_run.py:_restore():2185] restore
2024-02-03 00:44:35,036 INFO    MainThread:79316 [wandb_run.py:_restore():2191] restore done
2024-02-03 00:44:41,869 INFO    MainThread:79316 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-03 00:44:41,870 INFO    MainThread:79316 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-03 00:44:41,883 INFO    MainThread:79316 [wandb_run.py:_footer_sync_info():3816] logging synced files
